girder/api/v1/file.py

```diff
@@ -244,6 +244,8 @@ def readChunk(self, params):
                'so you should set it to the index of the byte one past the '
                'final byte you wish to receive.', dataType='integer',
                required=False)
+        .param('contentDisposition', 'Specify the Content-Disposition response '
+               'header disposition-type value', required=False)
```

Comment:
We should use the `enum` kwarg to this function to enumerate the allowed strings.